gapis/api/cmd_id_group.go

```diff
@@ -443,54 +442,68 @@ func (g CmdIDGroup) FindSubCommandRoot(id CmdID) *SubCmdRoot {
 	return nil
 }
 
-// AddAtoms fills the group and sub-groups with commands based on the predicate pred.
-//
-// If maxChildren is positive, the group, and any of it's decendent
-// groups, which have more than maxChildren child elements, will have their
-// children grouped into new synthetic groups of at most maxChildren children.
-//
-// If maxNeighbours is positive, we will group long list of ungrouped commands,
-// which are next to a group. This ensures the group is not lost in noise.
-func (g *CmdIDGroup) AddAtoms(pred func(id CmdID) bool, maxChildren, maxNeighbours uint64) error {
-	rng := g.Range
-	spans := make(Spans, 0, len(g.Spans))
-
-	scan := func(to CmdID) {
-		for id := rng.Start; id < to; id++ {
-			if !pred(id) {
-				rng.End = id
-				if rng.Start != rng.End {
-					spans = append(spans, &CmdIDRange{rng.Start, rng.End})
-				}
-				rng.Start = id + 1
+// AddCommand adds the command to the groups.
```

Comment:
I meant groups. It can propagate down to sub-groups. I admit I could expand on this a little more.